### ISSUE TITLE
problem in inference of custom trained model

### ISSUE THREAD

**Issue by amm266** [PaddlePaddle/PaddleOCR#10232]
- 系统环境/System Environment：win11 on CPU core i7 7700HQ
- 版本号/Version： Paddle：2.4.2  PaddleOCR：2.6 on GitHub
Hi, I've tried to train the paddle ocr model on a single photo to get used to its training pipeline and prepare for an actual project.
I've reached good results in training (acc = 0.99) but when I export the model and try its inference by the same picture I get irrelevant results.
[model config file](https://drive.google.com/file/d/1TXRRriaxAJXdWK6OIE97pRuwECjm-WaL/view?usp=sharing)
[train log](https://drive.google.com/file/d/1NbBjIimO7OhbTBSydUllCz0O8dtnpOHv/view?usp=sharing)
[char_dict_file]( https://drive.google.com/file/d/199NqVbU_qjlskClnbtbPb516fCWM45oF/view?usp=sharing)
[train image txt file](https://drive.google.com/file/d/1ywM4928HexjA3PA1y5B_4VBsFgNWqz87/view?usp=sharing)
picture: 
![img1](https://github.com/PaddlePaddle/PaddleOCR/assets/45390092/d168c498-be64-45cc-8f23-40a8523995b8)

train command: ```python tools/train.py -c configs/rec/PP-OCRv3/multi_language/kart_melly.yml```

export command:```python .\tools\export_model.py -c configs/rec/PP-OCRv3/multi_language/kart_melly.yml
[2023/06/23 00:07:31] ppocr INFO: train from scratch
[2023/06/23 00:07:34] ppocr INFO: inference model is saved to ./output/v3_kart_melly_inference\inference```

inference command:``` python tools/infer/predict_rec.py --image_dir="./train/dataset/img1.png" --rec_model_dir="output\v3_kart_melly_inference" --rec_char_dict_path="ppocr/utils/dict/word_dict_fa.txt" 
E0623 00:08:03.743561  8404 analysis_config.cc:110] Please use PaddlePaddle with GPU version.
[2023/06/23 00:08:03] ppocr INFO: In PP-OCRv3, rec_image_shape parameter defaults to '3, 48, 320', if you are using recognition model with PP-OCRv2 or an older version, please set --rec_image_shape='3,32,320
[2023/06/23 00:08:04] ppocr INFO: Predicts of ./train/dataset/img1.png:('ازاف', 0.0291864313185215)```

I am almost sure I am making a simple mistake but no matter how much I read the docs and search in issues I couldn't fix the problem.
thank's for your help.

**Comment by livingbody** [comment]
i cannot access  files on drive.google.com.
but i'm sure you need the correct char_dict_file to solve this problem.

**Comment by amm266** [comment]
> i cannot access files on drive.google.com. but i'm sure you need the correct char_dict_file to solve this problem.

I rechecked the links and all files are set to view for anyone with the link.
[https://drive.google.com/drive/folders/1bx37mhwWZmkdyANZeRC2pQXpdLVnfWzo?usp=drive_link](google drive folder)
in which command do you think char_dict_file is not set correctly?
I set it in the inference command and yml file to the same file.

**Comment by livingbody** [comment]
> > i cannot access files on drive.google.com. but i'm sure you need the correct char_dict_file to solve this problem.
> 
> I rechecked the links and all files are set to view for anyone with the link. [https://drive.google.com/drive/folders/1bx37mhwWZmkdyANZeRC2pQXpdLVnfWzo?usp=drive_link](google drive folder) in which command do you think char_dict_file is not set correctly? I set it in the inference command and yml file to the same file.

Please use [https://aistudio.baidu.com/aistudio/index](https://aistudio.baidu.com/aistudio/index) or pan.baidu.com and share the project or file with me。 I cannot access google products for some reasons...

**Comment by livingbody** [comment]
the problem is solved.
please give me five,  share the experience, and close the issue.

**Comment by livingbody** [comment]
## Problem Resolution

- Pre-trained models using English characters
- Use the pre-trained model that comes with the profile

**Comment by yimuchunshu** [comment]
how to solve,i encounter same problem

**Comment by tsufbechor** [comment]
??? Can you explain please how you solved this problem

**Comment by karanpate1** [comment]
i am facing same problem:

I fine-tuned the PP-OCRV5 on my custom dataset (containing English alphabets, numerals, and a few symbols).
-> I got a training accuracy of 98% and validation accuracy of 99%.
But I have an issue: when I ran the inference using my fine-tuned model, I am getting shitty results (even on images passed in the training and validation dataset).
-> Can you explain what could be a probable reason?
->i followed all the steps like evaluation and model export as documentation
